lib/deref.js

```diff
@@ -232,13 +232,20 @@ module.exports = {
       schema.default = '<object>';
     }
     else if (schema.type === 'array' && schema.items) {
-
-      // For VALIDATION - keep minItems and maxItems properties defined by user in schema as is
-      // FOR CONVERSION - need to set both properties to 2 for schema faking
+      /*
+        For VALIDATION - keep minItems and maxItems properties defined by user in schema as is
+        FOR CONVERSION - set both properties as minimum and valid as possible for schema faking (to lessen faked items)
+          We have enforced limit to maxItems as 100, set by Json schema faker option
+      */
       if (resolveFor === 'CONVERSION') {
-        // This is needed because the schemaFaker doesn't respect options.maxItems/minItems
-        schema.maxItems = 2;
-        schema.minItems = 2;
+        // Override maxItems to default (2) if no minItems present
```

Comment:
Should this comment be "Override minItems to 2..."?

Reply:
Also I feel `if` statements are simpler to read.

Reply:
@umeshp7 Yeah, it should be also i'll update statements to if.